### PR TITLE
V1 2 3

### PR DESCRIPTION
**Pull request by elfisher**
### Description
Replaces https://github.com/opensearch-project/project-website/pull/590 and includes DCO for updated Logstash version.

- Adds artifact and version metadata for v1.2.3
- Updates blog post 1.2.3

 
### Issues Resolved
opensearch-project/OpenSearch#1734

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _versions/2021-12-22-opensearch-1.2.3.markdown [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    artifact: opensearch-cli
+    version: 1.1.0
+  -
+    role: ingest
+    artifact: logstash-oss-with-opensearch-output-plugin
+    version: 7.16.2
+  -
+    role: ingest
+    artifact: data-prepper
+    version: data-prepper-1.2.0
```
Comment (elfisher) [inline _versions/2021-12-22-opensearch-1.2.3.markdown:28]: 1.2.1

**Review by elfisher — COMMENTED** [review]
A few changes.

File: _artifacts/opensearch-1.2/x86/_opensearch-1.2.3-freebsd-x86.markdown [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+artifact_id: opensearch
+version: 1.2.3
+platform: freebsd
+architecture: x86
+slug: opensearch-1.2.3-freebsd-x86
+category: opensearch
+type: system-package
+freebsd_package_name: opensearch
+link: https://www.freshports.org/textproc/opensearch
+---
```
Comment (dblock) [inline _artifacts/opensearch-1.2/x86/_opensearch-1.2.3-freebsd-x86.markdown:12]: FreeBSD version hasn't been released yet, remove.

File: _posts/2021-12-16-update-1-2-2.markdown [hunk 3]
```diff
@@ -11,7 +11,11 @@ excerpt: "As indicated in the previous blog post, CVE-2021-45046 was issued shor
 
 ---
 
+### Update 2021-12-22:
 
+OpenSearch version 1.2.3 is now available. This release updates the version of Log4j used in OpenSearch to Log4j 2.17.0 as recommended by the advisory in [CVE-2021-45105](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45105). Additionally, this release addresses a [regression introduced in OpenSearch 1.2.0](https://github.com/opensearch-project/OpenSearch/issues/1734) for those using repository-azure plugin. All users of OpenSearch should update to OpenSearch 1.2.3. [Learn more](/blog/releases/2021/12/update-1-2-3/).
+
+### Update your clusters to 1.2.2
```
Comment (dblock) [inline _posts/2021-12-16-update-1-2-2.markdown:18]: Update to 1.2.3 and mention both CVEs.

File: _versions/2021-12-22-opensearch-1.2.3.markdown [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    version: 1.2.3
+  -
+    role: ui
+    artifact: opensearch-dashboards
+    version: 1.2.0
+  -
+    role: command-line-tools
+    artifact: opensearch-cli
+    version: 1.1.0
+  -
```
Comment (dblock) [inline _versions/2021-12-22-opensearch-1.2.3.markdown:21]: Don't include things that haven't been released.

File: _versions/2021-12-22-opensearch-1.2.3.markdown [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        explanation: "downloads/data-prepper.html"
+      logstash-oss-with-opensearch-output-plugin:
+        explanation: "downloads/logstash-oss-with-opensearch-output-plugin.markdown"
+  command-line-tools:
+    role: command-line-tools
+    artifacts:
+      opensearch-cli:
+        explanation: "downloads/opensearch-cli.html"
+
+
```
Comment (dblock) [inline _versions/2021-12-22-opensearch-1.2.3.markdown:70]: There's no dashboards release.

**Review by dblock — CHANGES_REQUESTED** [review]

File: _posts/2021-12-16-update-1-2-2.markdown (hunk 3 above)
Comment (elfisher) [reply _posts/2021-12-16-update-1-2-2.markdown:18]: This is the old blog. Shortening text to be more clear.

File: _artifacts/opensearch-1.2/x86/_opensearch-1.2.3-freebsd-x86.markdown (hunk 2 above)
Comment (elfisher) [reply _artifacts/opensearch-1.2/x86/_opensearch-1.2.3-freebsd-x86.markdown:12]: done

File: _versions/2021-12-22-opensearch-1.2.3.markdown (hunk 5 above)
Comment (elfisher) [reply _versions/2021-12-22-opensearch-1.2.3.markdown:70]: no new dashboards release is included.

**Review by AMoo-Miki — DISMISSED** [review]

File: _posts/2021-12-16-update-1-2-2.markdown [hunk 6]
```diff
@@ -13,7 +13,7 @@ excerpt: "As indicated in the previous blog post, CVE-2021-45046 was issued shor
 
 ### Update 2021-12-22
 
-OpenSearch version 1.2.3 is now available and addresses [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228), [CVE-2021-45046](https://nvd.nist.gov/vuln/detail/CVE-2021-45046), [CVE-2021-45105](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45105). Update your clusters to 1.2.3. [See the release blog](/blog/releases/2021/12/update-1-2-3/).
+OpenSearch version 1.2.3 is now available and addresses [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228), [CVE-2021-45046](https://nvd.nist.gov/vuln/detail/CVE-2021-45046), and[CVE-2021-45105](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45105). Update your clusters to 1.2.3. [See the release blog](/blog/releases/2021/12/update-1-2-3/).
```
Comment (dblock) [inline _posts/2021-12-16-update-1-2-2.markdown:16]: Missing a space.

**Review by dblock — DISMISSED** [review]

**Review by dblock — APPROVED** [review]

**Review by bbarani — APPROVED** [review]

**Review by peterzhuamazon — APPROVED** [review]
Thanks.

**Review by AMoo-Miki — APPROVED** [review]

**Pull request merged**